packages/vivaldi-deb/vivaldi-deb.pacscript

```diff
@@ -0,0 +1,6 @@
+name="vivaldi-deb"
+version="4.0.2312.41-1"
+url="https://downloads.vivaldi.com/stable/vivaldi-stable_${version}_amd64.deb"
+description="An advanced browser made with the power user in mind."
+hash="70f552d36b520cbfe747b24902a968ad03c359ffa49882869a60f03ca6c131bc"
```

Comment:
It also needs a `gives` field

Se [`brave-browser-beta`](https://github.com/pacstall/pacstall-programs/blob/master/packages/brave-browser-beta/brave-browser-beta.pacscript)